packages/headless/src/api/apolloClient.ts

```diff
@@ -15,18 +15,14 @@ import { CookieOptions, getAccessToken } from '../auth';
 
 export type PersistentContext = Record<string, unknown>;
 
-let WP_URL = trimTrailingSlash(
+const WP_URL = trimTrailingSlash(
   process.env.NEXT_PUBLIC_WORDPRESS_URL || process.env.WORDPRESS_URL,
 );
 
 if (!WP_URL) {
-  WP_URL = 'https://headlessfw.wpengine.com';
```

Comment:
@wjohnsto I tweaked the docs in https://github.com/wpengine/headless-framework/pull/167/commits/5b8ae2bdda01eee7730b74d92ac4c48dda61b9ee to account for the removal of the fallback URL here.

It just prompts users to create the env file before running `npm run dev` for the first time, otherwise there's no WP_URL and all pages 404. Hope that's ok!

This is otherwise working well for me.